doc/releases/v0_11_2.md

```diff
@@ -17,6 +17,7 @@ Bug Fixes
   linear and angular base velocity, described in https://github.com/robotology/idyntree/issues/370 .
 * Fixed compilation on 32-bit Windows ( https://github.com/robotology/idyntree/pull/506 ).
 * Fixed a URDF parser regression introduced in iDynTree 0.11 in URDF parser for which collision geometeries were also loaded as visual geometries, and viceversa (https://github.com/robotology/idyntree/issues/497, https://github.com/robotology/idyntree/pull/559).
+* Fixed a URDF parser regression introduced in iDynTree 0.11 in URDF parser for which geometries element without origin tag were not correctly parsed (https://github.com/robotology/idyntree/issues/496, https://github.com/robotology/idyntree/pull/564).
```

Comment:
Fixed a URDF parser regression introduced in iDynTree 0.11 ~~in URDF parser~~ for which *geometry elements* without `origin` tag were not correctly parsed